### PR TITLE
Create CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by jshcodes**

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+```shell
+COVERAGE RESULTS SHOULD BE POSTED HERE
+```
+
+## Added features and functionality
++ Details of new features and functionality should be listed here
+
+## Issues resolved
++ Bug fix: Bugs should be listed individually. Please link your PR to any related issues so that they are triaged properly. Add a link to this line to the issue.
```
Comment (shawndwells) [inline .github/pull_request_template.md:22]:
```suggestion
+ Bug fix: Please list related bugs individually. Identifying the ticket or bug report in the PR description auto-updates the affected ticket and helps the community with ticket triage. For example:

* Fixes https://github.com/CrowdStrike/falconpy/issues/1234 by doing foo 
* Mitigates https://github.com/CrowdStrike/falconpy/issues/5678 by updating bar.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
+# Contributing to this repository <!-- omit in toc -->
+
+## Getting started <!-- omit in toc -->
+_Welcome!_ We're excited you want to take part in the FalconPy community! 
+
+Please review this document for details regarding project needs, contributor requirements and our pull request process. If you have any questions, please let us know by
```
Comment (shawndwells) [inline CONTRIBUTING.md:6]:
```suggestion
Please review this document for details regarding getting started with your first contribution, packages you'll need to install as a developer, and our pull request process. If you have any questions, please let us know by
```

File: .github/pull_request_template.md (hunk 1 above)
Comment (shawndwells) [reply .github/pull_request_template.md:22]: * Saying please first is just polite!
* Gave examples of how to call out tickets

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+posting in your question in the [discussion board](https://github.com/CrowdStrike/falconpy/discussions).
+
+### Before you begin
+- Have you read the [Code of Conduct](CODE_OF_CONDUCT.md)?
```
Comment (shawndwells) [inline CONTRIBUTING.md:10]:
```suggestion
- Have you read the [Code of Conduct](CODE_OF_CONDUCT.md)? The Code of Conduct helps us establish community norms and how they'll be enforced.
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+
+### Table of Contents
+- [How you can contribute](#how-you-can-contribute)
+- [Pull Requests](#pull-requests)
+- [Suggestions](#suggestions)
+
+## How you can contribute
+- Submit a [bug report](https://github.com/CrowdStrike/falconpy/issues).
```
Comment (shawndwells) [inline CONTRIBUTING.md:18]:
```suggestion
- See something? Say something! Submit a [bug report](https://github.com/CrowdStrike/falconpy/issues) to let the community know what you've experienced or found. Bonus points if you suggest possible fixes or what you feel may resolve the issue. For example: "_Attempted to use the XZY API class but it errored out. Could a more descriptive error code be returned?_"
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+If this is a concern, contact the maintainers before contributing.
+
+### Additional contributor package requirements
+The file `requirements-dev.txt` contains additional Python requirements used for developing unit tests and for accessing the integrated debugger within FalconShell.
+Review this file's contents and install these requirements before proceeding:
++ flake8
++ pytest
++ pytest-cov
++ coverage
++ iPython
```
[the remark this answers is not in the document]
Comment (jshcodes) [reply CONTRIBUTING.md:48]: Not really, just being over-verbose. :smile:

File: requirements-dev.txt [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+# To update, run:
+#
+#    pip-compile requirements-dev.in
+#
+# These packages are not necessary in order to 
+# communicate with the CrowdStrike Falcon OAuth2 API.
+#
+# These packages are only required if you intend
+# on using FalconShell, or are a contributor and are
+# developing unit tests for the project.
```
[the remark this answers is not in the document]
Comment (jshcodes) [reply requirements-dev.txt:12]: Doesn't bother me, and I like the legibility. :smile:

File: .github/pull_request_template.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+- [ ] Major Feature update
+- [ ] Bug fixes 
+- [ ] Breaking Change
+- [ ] Updated unit tests
+- [ ] Additional documentation required
+
+> Check the values above that match your PR and remove the remaining.
+
+```shell
+COVERAGE RESULTS SHOULD BE POSTED HERE
```
[the remark this answers is not in the document]
Comment (jshcodes) [reply .github/pull_request_template.md:15]: Absolutely, and an excellent suggestion.  Will add this before this merges.

[the remark this answers is not in the document]
Comment (jshcodes) [reply .github/pull_request_template.md:15]: CONTRIBUTING.md and pull_request_template.md both have been updated to reflect this addition.

**Pull request merged**